beetsplug/discogs.py

```diff
@@ -270,12 +270,17 @@ def get_album_info(self, result):
         # `autotag.apply_metadata`, and set `medium_total`.
         for track in tracks:
             track.media = media
-            track.medium_total = mediums
+            track.medium_total = mediums.count(track.medium)
+            # Invent our own track IDs as Discogs doesn't have them
+            track.track_id = str(album_id) + "-" + track.track_alt
```

Comment:
As proposed in https://github.com/beetbox/beets/issues/2336#issuecomment-384488651

Reply:
Should I add a full URL to that particular comment or only mention the issue number?

Reply:
Just the issue number is fine.